packages/app-project/src/shared/components/AboutProject/AboutProject.js

```diff
@@ -0,0 +1,34 @@
+import counterpart from 'counterpart'
+import { Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import en from './locales/en'
+import ContentBox from '../ContentBox'
+import styled from 'styled-components'
+
+counterpart.registerTranslations('en', en)
+
+const StyledParagraph = styled(Paragraph)`
+  max-width: 100%;
+`
+
+function AboutProject ({ projectName, description }) {
+  const aboutProjectTitle = `${counterpart(
```

Comment:
counterpart can interpolate strings, which is needed for translating right-to-left languages. There's an example at https://github.com/martinandert/counterpart#interpolation

Reply:
Here's the fix for this one: https://github.com/zooniverse/front-end-monorepo/pull/660/commits/d46ee0721d3863ed2d282f81c62e89d55534dd1d